app/views/boxes/index.haml

```diff
@@ -0,0 +1,44 @@
+= render "filters"
+
+.row
+  .col
+    - if @boxes.empty?
+      = empty_data icon: "icon-outline-box xx-large" do |c|
+        - c.body do
+          %h1 There are no boxes
+          %p Every box can hold multiple samples
+    - else
+      =form_tag bulk_action_boxes_path, id: "boxes_form", method: :post do
+        = cdx_table title: pluralize(@total, "box") do |t|
+          - t.actions do
+            =button_tag id: "bulk_destroy", class: "btn-link", name: "bulk_action", value: "destroy", data: { method: "delete", confirm: "You're about to permanently delete the selected batches. This action CANNOT be undone. Are you sure you want to proceed?" } do
+              .icon-trash.btn-icon
+          - t.columns do
+            %col{:width => "5%"}
+            %col{:width => "30%"}
+            %col{:width => "50%"}
+            %col{:width => "15%"}
+          - t.thead do
+            %tr
+              %th
+              %th Box Id
+              %th Purpose
+              %th Samples
+          - t.tbody do
+            - @boxes.count_samples.each do |box|
+              %tr.laboratory-sample-row{data: {href: box_path(box)}}
+                %td
+                  =check_box_tag "box_ids[]", box.id, false, { id: "box_ids.#{box.id}" }
+                  %label.row{for: "box_ids.#{box.id}"}
+                %td= box.uuid
```

Comment:
We should probably use the short format for UUID.
```suggestion
                %td= short_uuid_with_title box.uuid
```

Reply:
Full ID when there is room. No doubt.  Strong opinion here.  :)

Reply:
IMO it shouldn't be about available space but about what's important. If the full UUID is important, we should make it happen that we can show it everywhere in full. If it's not important and showing the shortened version is good, then we should go with that everywhere. Not just because of consistency between different displays, but also because a shorter UUID is easier on the eye of the viewer than a long string of characters that have little informative value.

Reply:
Having the long ID isn't important, but it could prove to be useful.  For instance, it could be easier to copy the full ID, and then search/paste it.  Plus other use cases that we might not imagine.  I see your point that it would be easier for the human eye to read the shortened version.  Both have a contra, but one of them is about making something not as nice, and the other one is about making something impossible.  That's why I prefer to avoid the impossible situation, when there are no other drawbacks (when we add not having enough room in the picture, then the balance turns in favor of the shortened version).  Of course this is subjective, but this is my view.

Reply:
>  For instance, it could be easier to copy the full ID, and then search/paste it. Plus other use cases that we might not imagine.

If we need the copy functionality or any other feature, we'll also need it in places where there's less available space. Thus we should implement a UI that works in these situations.

Reply:
Nobody asked for being able to copy the UUID.  So there is no need per se.  But nothing prevents users from doing so, right? Now, this doesn't mean that we have to be prepared for this situation in every page.  That would be crazy, as nobody asked for it, there's no real use case.  However, we know that users can manually copy anything. If we can easily improve their experience at least in 1 page, without having a negative impact in something else, why wouldn't we do it?  Is it just for being consistent? Is it really inconsistency when the diff is based on logic and not on randomness?

I feel I'm discussing too much about a not-important detail, and based on my imagination instead of actual user feedback.  

I'm willing to let it go, so go ahead with the shortened version.